pkg/processor/meta.go

```diff
@@ -16,6 +16,7 @@ const metaTemplate = `apiVersion: %[1]s
 kind: %[2]s
 metadata:
   name: %[3]s
+%[7]s
```

Comment:
is there will be empty line if no namespace provided?
can you please regenerate chart examples and push changes to your branch with
```cat test_data/sample-app.yaml | go run ./cmd/helmify examples/app```
to see that output has not changed with disabled flag?

Reply:
> is there will be empty line if no namespace provided? can you please regenerate chart examples and push changes to your branch with `cat test_data/sample-app.yaml | go run ./cmd/helmify examples/app` to see that output has not changed with disabled flag?

Hi @arttor ,
Apologies for the late reply.
I ran the command `cat test_data/sample-app.yaml | go run ./cmd/helmify examples/app` and it does not make any changes to the existing generated files.
I believe the lines 
`metaStr = strings.Trim(metaStr, " \n") metaStr = strings.ReplaceAll(metaStr, "\n\n", "\n")` are making sure that there is no empty line left.

I can see the desired output when I run the command `cat test_data/sample-app.yaml | go run ./cmd/helmify -preserve-ns  examples/app/preservens `

I am not sure if I should commit all these files since this will make the repository bulky.
Please let me know the next steps.

Reply:
[preservens.zip](https://github.com/arttor/helmify/files/15381288/preservens.zip)
I am including the generated chart with `cat test_data/sample-app.yaml | go run ./cmd/helmify -preserve-ns  examples/app/preservens`